frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -252,6 +259,54 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
+	/// Greedily reduce the size of the solution to fit into the block w.r.t length.
+	///
+	/// The length of the solution is largely a function of the number of voters. The number of
+	/// winners cannot be changed, and the `ElectionSize` is a proxy for the total number of stakers.
```

Comment:
```suggestion
	/// winners cannot be changed, and the `ElectionSize` is a proxy for the total number of stakers.
```

what is `ElectionSize`? copy pasta? :D

Reply:
Er, yes.

Reply:
https://github.com/paritytech/substrate/pull/8317/commits/d05ce23bb1bca9543280e88246026264635ce112